### PR TITLE
Explicitly set --cluster-ip-range --clean-start --minStartupPods

### PR DESCRIPTION
**Pull request by fejta**
ref https://github.com/kubernetes/test-infra/pull/3535 https://github.com/kubernetes/test-infra/pull/3375

Also remove unused `GINKGO_PARALLELISM`

/assign @krzyzacy @ixdy

### CODE REVIEW

File: hack/ginkgo-e2e.sh [hunk 1]
```diff
@@ -38,12 +38,6 @@ GINKGO_NO_COLOR=${GINKGO_NO_COLOR:-n}
 # If 'y', will rerun failed tests once to give them a second chance.
 GINKGO_TOLERATE_FLAKES=${GINKGO_TOLERATE_FLAKES:-n}
 
-# The number of tests that can run in parallel depends on what tests
-# are running and on the size of the cluster. Too many, and tests will
-# fail due to resource contention. 25 is a reasonable default for a
-# 3-node (n1-standard-1) cluster running all fast, non-disruptive tests.
-GINKGO_PARALLELISM=${GINKGO_PARALLELISM:-25}
```
Comment (krzyzacy) [inline hack/ginkgo-e2e.sh]: unused?

Comment (fejta) [inline hack/ginkgo-e2e.sh]: Yes, unused.

Looks like this was replaced with `GINKGO_PARALLEL_NODES` and `GINKGO_PARALLEL`

**Comment by krzyzacy** [comment]
LGTM

and let https://github.com/kubernetes/test-infra/pull/3535 go in first

**Pull request merged**